scipy/special/tests/test_kolmogorov.py

```diff
@@ -237,10 +237,10 @@ def test_oneovernclose(self):
         dataset0 = np.column_stack([n, x, pp])
         FuncData(_smirnovp, dataset0, (0, 1), 2, rtol=_rtol).check(dtypes=[int, float, float])
 
-        x = 1.0/n + 2*np.finfo(float).eps
+        x = 1.0/n + 2*np.finfo(np.float32).eps
```

Comment:
does the test really fail without this line but with the 1e-5 rtol? it's probably the `int` that's the problem, that'll be int32 on 32-bit platform - `float` on the other hand is always float64.

Reply:
Yes, it really fails in that scenario; I tried a bunch of permutations on the types / tolerances to produce a minimally invasive fix.